modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -951,10 +951,16 @@ CV_ALWAYS_INLINE void run_arithm_s(Buffer &dst, const View &src, const float sca
         break;
     }
     case ARITHM_DIVIDE:
-        for (int w=0; w < width; w++)
-            for (int c=0; c < chan; c++)
-                out[chan*w + c] = div<DST>(in[chan*w + c], scalar[c], scale);
+    {
+        int w = 0;
+#if CV_SIMD
+        //w = divc_simd(in, scalar, out, length, chan, scale);
```

Comment:
it seems no SIMD at now, then may be PR description should be modified like: "Prepare SIMD for DIVC"?

Reply:
It's line from old revision. Please take a look fresh version ;-)